floss/language/go/extract.py

```diff
@@ -18,6 +18,50 @@
 MIN_STR_LEN = 6
 
 
+def calculate_coverage(enhanced_static_strings: List[StaticString], static_strings: List[StaticString]) -> float:
```

Comment:
I'm getting `110.91%` for `072e3cc7477f29287f0625b8f22d2d82a66407553859ec99c719087a680125ab`, so something seems off :)